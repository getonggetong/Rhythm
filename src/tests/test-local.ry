def c;

foo(c) {
	c--;
	print(c);
	bar();
}

bar() {
	print(c);
}


init() {
	c = C4.4;
	foo(c);
}