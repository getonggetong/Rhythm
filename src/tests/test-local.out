B4.4
C4.4